src/EncompassRest/EncompassRestClient.cs

```diff
@@ -15,6 +15,13 @@ namespace EncompassRest
 {
     public sealed class EncompassRestClient : IDisposable
     {
+#if !HAVE_SSL_PROTOCOLS
+        static EncompassRestClient()
+        {
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
```

Comment:
add NOT's for Tls 1.0 and 1.1? to ensure security?

Reply:
It's a global setting so I don't want to force all applications to use TLS 1.2 when other services users consume might not support TLS 1.2.

Reply:
ah. ok.